src/WebDocs/wwwroot/app/menu/0000 - Drapo.html

```diff
@@ -6,6 +6,43 @@
+        
+        <p>Drapo is a powerful, declarative frontend framework that brings reactive programming to .NET web applications without the complexity of traditional JavaScript frameworks. It uses HTML attributes to create dynamic, data-driven user interfaces with seamless server-side integration.</p>
+        
+        <h3>Key Features</h3>
+        <ul>
+            <li><strong>Declarative Data Binding:</strong> Two-way data binding using simple HTML attributes like <code>d-model</code></li>
+            <li><strong>Template System:</strong> Dynamic content rendering with <code>d-for</code> loops and <code>d-if</code> conditions</li>
+            <li><strong>AJAX Integration:</strong> Built-in data loading and posting with <code>d-dataUrlGet</code> and <code>d-dataUrlSet</code></li>
+            <li><strong>Real-time Communication:</strong> Live data updates using <code>d-dataPipes</code> for real-time features</li>
+            <li><strong>Event Handling:</strong> Rich event system with <code>d-on-*</code> attributes for user interactions</li>
+            <li><strong>Form Validation:</strong> Integrated validation system with custom rules and error handling</li>
+            <li><strong>Component Architecture:</strong> Reusable components and sectoring for modular applications</li>
+            <li><strong>Storage Management:</strong> Client-side data storage with automatic change tracking</li>
+        </ul>
+        
+        <h3>Why Choose Drapo?</h3>
+        <ul>
+            <li><strong>No JavaScript Required:</strong> Build complex SPAs using only HTML attributes and .NET</li>
+            <li><strong>Seamless .NET Integration:</strong> Works naturally with ASP.NET Core and existing .NET infrastructure</li>
+            <li><strong>Rapid Development:</strong> Declarative approach reduces boilerplate code significantly</li>
+            <li><strong>Learning Curve:</strong> Familiar to .NET developers, no need to learn complex JavaScript frameworks</li>
+            <li><strong>Performance:</strong> Optimized for .NET applications with efficient data binding and rendering</li>
+        </ul>
+        
+        <h3>Perfect For</h3>
+        <ul>
+            <li>Enterprise web applications requiring rich interactivity</li>
+            <li>Dashboard and admin interfaces with real-time data</li>
+            <li>Form-heavy applications with complex validation</li>
+            <li>.NET teams wanting to avoid JavaScript framework complexity</li>
+            <li>Applications requiring tight integration with .NET backend services</li>
+        </ul>
+        
+        <p>Get started with Drapo by exploring the <a href="#" onclick="ExecuteDataItem({{item.Action}},item in menuItems,{{item.Name}}=='Installation',false,item in item.Items)">Installation</a> guide and discover how to build modern, interactive web applications with the power and familiarity of .NET.</p>
```

Comment:
Here you must use the router function ApplyRoute to go installation.  this function must be used with a d-on-click